src/LinuxTracing/UprobesReturnAddressManagerTest.cpp

```diff
@@ -401,15 +401,15 @@ namespace {
 
 const std::string maps_string =
     "55d0f260c000-55d0f260d000 r--p 00000000 fe:00 3415204                    "
-    "/usr/local/uprobes_target\n"
+    "/usr/local/target_fp\n"
```

Comment:
I know we don't often do it, but could we add the sources and instructions on how to build `target_fp` and `target_no_fp` to `testdata`? I'm always quite lost having no idea what's inside these binaries.

Reply:
Added the .cc file into testdata with instructions how to build.